### ISSUE TITLE
[DownView] Support for internal links?

### ISSUE THREAD

**Issue by antonjazz** [iwasrobbed/Down#93]
Please help prevent duplicate issues before submitting a new one:

* [x] I've searched other open/closed issues for duplicates before opening up this new issue.

# Report

## What did you do?

I'm trying to use Down on markdown with internal links. An example is this [link](#destinationtext) to text at the bottom of this page.

I tried multiple standard ways of specifying the link and destination anchor, including:

- `[link text](#anchorname)` and `<a name="anchorname"></a>` \
(This complies with the CommonMark spec so it should work, right?)
- `[link text](#section-name)` and `## Section Name` \
(This is a common standard when linking to a section, though it's not part of CommonMark standard.)

## What did you expect to happen?

I expected the links to function. They function here and the first one functions on the [commonmark dingus](https://spec.commonmark.org/dingus/).

## What happened instead?

The links did nothing.

---
<a name="destinationtext">PS</a> here is the destination of the link I used as an example earlier on.

**Comment by iwasrobbed** [comment]
Happy to accept a pull request with this feature 👍

**Comment by antonjazz** [comment]
Ah, I wish I were at that point! I'm brand new to github and almost brand new to iOS.
I appreciate the work you're doing and I look forward to the time when I'll be able to help contribute!

As far as this issues goes, I was surprised to find that not even  
`<a href="#anchor">linking text</a>` and `<a name="anchor">linked-to text</a>`  
worked when I ran it through Down. Since that's just standard HTML, which I thought was passed through unchanged, I figured that there was some special link processing going on that you might be able to tell me about… I guess not?

Thank you, in any case—for this and for the great cocoapod!

Anton
